docker/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
```

Comment:
I created this docker-compose set up so that I could easily debug tests that interact with the database. 

The existing setup by default ran all the tests from **inside a Docker container**, which leaves no way to attach a debugger. This setup just runs PG as a service exposed on port 5432 of the host, which is what the configuration for local testing expects